pkg/utils/test/integration.go

```diff
@@ -29,7 +29,7 @@ const (
 	BootstrapTestEnvRetries  = 1
 
 	// CRDsRelativePath kubebuilder 2.3.1 comes with a 1.15 control plane
```

Comment:
Either this comment is no longer true or something will break during integration tests.

Reply:
I don't know how this has been true in the past. Kubebuilder [`2.3.1`](https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_linux_amd64.tar.gz) uses `Kubernetes v1.16.4`. I'll remove this comment.